applications/ShallowWaterApplication/custom_processes/depth_integration_process.h

```diff
@@ -181,6 +181,8 @@ class KRATOS_API(SHALLOW_WATER_APPLICATION) DepthIntegrationProcess : public Pro
 
     void Integrate(PointerVector<GeometricalObject>& rObjects, NodeType& rNode);
 
+    array_1d<double,3> InterpolateVelocity(const int ElementId, const Vector& rShapeFunctionValues) const;
```

Comment:
```suggestion
    array_1d<double,3> InterpolateVelocity(
        const int ElementId,
        const Vector& rShapeFunctionValues) const;
```
Minor style guide suggestion.